### PR TITLE
Fix all of the obvious and easy to fix stuff I ran into during my review

### PR DESCRIPTION
**Pull request by Enduriel**

### CODE REVIEW

File: msu/systems/mod_settings/load.nut [hunk 1]
```diff
@@ -92,15 +90,16 @@ local testSettingsSystem = function()
 			"NewCampaignOnly"
 		]
 	}
-}
+};
```
Comment (LordMidas) [inline msu/systems/mod_settings/load.nut:93]: No semi-colon at the end of function definitions

File: msu/utilities/math.nut [hunk 2]
```diff
@@ -5,8 +5,8 @@
 		local count = 0;
 		while (_num > 1) 
 		{
-		    _num = _num >> 1;
-		    count++;
+			_num = _num >> 1;
```
Comment (LordMidas) [inline msu/utilities/math.nut:8]: Why double tab?

File: ui/mods/msu/utilities.js [hunk 3]
```diff
@@ -93,5 +93,5 @@ MSU.getLocalString = function( _key, _value, _maxLen, _depth, _isArray )
 
 MSU.capitalizeFirst = function ( _string )
 {
-	return _string.charAt(0).toUpperCae() + _string.slice(1);
+	return _string.charAt(0).toUpperCase() + _string.slice(1);
```
Comment (LordMidas) [inline ui/mods/msu/utilities.js:96]: LOL

**Review by LordMidas — CHANGES_REQUESTED** [review]

File: msu/systems/mod_settings/load.nut (hunk 1 above)
Comment (Enduriel) [reply msu/systems/mod_settings/load.nut:93]: That's not a function

File: msu/config/character.nut [hunk 4]
```diff
@@ -37,23 +37,21 @@
...
+		});
 
 		if (_damageTypeName = "")
 		{
 			_damageTypeName = _damageType;
 		}
 
 		::Const.Damage.DamageTypeName.push(_damageTypeName);
-	}
+	};
```
[the remark this answers is not in the document]
Comment (Enduriel) [reply msu/config/character.nut:54]: This is right idk what I was doing

File: msu/hooks/skills/skill.nut [hunk 5]
```diff
@@ -142,14 +142,14 @@
 	}
 
 	local setContainer = o.setContainer;
-	o.setContainer = function( _c )
+	o.setContainer = function( _container )
```
[the remark this answers is not in the document]
Comment (Enduriel) [reply msu/hooks/skills/skill.nut:145]: Bruh why does vanilla need to be dumb sometimes

File: msu/utilities/math.nut (hunk 2 above)
Comment (Enduriel) [reply msu/utilities/math.nut:8]: wdym double tab? it's aligned properly, was just spaces before instead of tabs

**Pull request merged**